### DISCUSSION TITLE
What's the easiest way to get a "--from-nodes" "--base-env" sort of experience?

### DISCUSSION THREAD

**Discussion by pascalwhoop** [kedro-org/kedro#4020] [Idea]
Often you want to run the entire pipline in env A but start reading the first datasets from env B

![image](https://github.com/user-attachments/assets/bb1c8a33-7282-4b25-bd68-c4ad452a7e50)

E.g. you want to run in Dev but the root datasets should be in prod. When developing we often find ourselves trying to get Prod working and it crashes in K8S/Argo/... Then it's quite tedious to keep working there, you want to move back to your machine. But how do you say

> run from node X and read the input data from Prod but then keep going in Dev

? This would be super helpful I think. Right now I copy the datasets over manually from Prod to Dev and then keep workin in Dev. But it would be neat to say `--read-from-env prod` and this reads the first entrypoints from prod, then keeps going in the env chosen.

**Comment by noklam** [comment]
@pascalwhoop Maybe I don't understand the question fully. Can you have a duplicate dataset in `prod/dev`?
